src/Models/VectorStructInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Qdrant\Models;
+
+interface VectorStructInterface
```

Comment:
I like `VectorStructInterface` to implement both `VectorStruct` and  `MultiVectorStruct`.